themes/default/layouts/partials/registry/package/breadcrumb.html

```diff
@@ -1,29 +0,0 @@
-<!-- Based on https://gohugo.io/content-management/sections/#example-breadcrumb-navigation -->
-
-<ol class="docs-breadcrumb">
-    {{ template "registrybreadcrumbnav" (dict "p1" . "p2" .) }}
-</ol>
-
-{{ define "registrybreadcrumbnav" }}
```

Comment:
some of this code was dropped and needs to be added to the breadcrumb component over in hugo; as its accounting for things like the api docs issue